cmd/root.go

```diff
@@ -39,7 +40,7 @@ func initConfig() {
 
 	// Try to read config
 	if err := viper.ReadInConfig(); err != nil {
-		PrintErrorAndExit(err)
```

Comment:
Why was this change introduced? My thought was to fail as early as possible. However, I see that your message is better.